application.py

```diff
@@ -355,5 +372,6 @@ def movie_option(userID):
 
 # Run server
 if __name__ == '__main__':
-    # application.run(host='0.0.0.0')
+
+    #application.run(host='0.0.0.0')
```

Comment:
I believe we should not comment this out as the server needs this line to run. Since we want to have the master branch represent the latest shippable code we should uncomment this.

Reply:
Thanks for catching this.  I have committed the changes you have suggested. Should be good to deploy now.